DigitalLearningSolutions.Web/ViewModels/Common/ViewComponents/NumberOfAdministratorsViewModel.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents
 {
-    using DigitalLearningSolutions.Data.Models;
-    using DigitalLearningSolutions.Web.Helpers;
+    using global::DigitalLearningSolutions.Data.Models;
+    using global::DigitalLearningSolutions.Web.Helpers;

```

Comment:
Think this is a knock-on effect from the weird namespaces. This file shouldn't need any changes.